reports/Ohio.md

```diff
@@ -1,3 +1,16 @@
+## Canton
+
+### Police spray mace, fire tear gas and flash bangers at protestors, even after they have dispersed | May 29th
+
+Police warn protestors blocking road that chemical agents will be deployed. The crowd mostly disperses after the warning, but police start using mace anyway. Police continue to use tear gas, flash bangers, and mace, even after protestors are out of the roadway. In part 2 of the video at 7:58, one of the used flash bang canisters is shown. It is labeled 5440 Launchable Flameless Expulsion OC.
+
+id: oh-canton-1
```

Comment:
All the record ids are same `oh-canton-1`, should they be different for each of the data set?

Reply:
Yeah these are okay because they're all the first record in each city, I believe